### PR TITLE
Document Environment Variables

### PR DESCRIPTION
**Pull request by TheoLechemia**
Small contribution on documentation.
See https://github.com/flask-admin/flask-admin/issues/1694
No sure if it's the way you want to do it.. tell me

### CODE REVIEW

**Comment by samuelhwilliams** [comment]
Thanks for the suggested improvement to document env var configuration on Flask-Admin; I agree this is something we're missing at the moment.

We've just removed env var configuration for theming (swatch, fluid layout) and move it to a Theme config object passed to the admin instance. It looks like we still have two possible env vars: `ADMIN_RAISE_ON_INTEGRITY_ERROR` and `ADMIN_RAISE_ON_VIEW_EXCEPTION`. It would be good to add documentation for those. I think it makes sense to flag these at the bottom of 'introduction.rst' rather than hiding them in 'advanced.rst'.

If you wanted to update the PR I'd be very grateful - if not I'll try to get around to it soon.

File: doc/advanced.rst [hunk 1]
```diff
@@ -545,3 +545,20 @@ While the wrapped function should accept only one parameter - `ids`::
...
 
                 flash(gettext('Failed to approve users. %(error)s', error=str(ex)), 'error')
+
+
+Settings reference
+------------------
+
+This section list settings supported in the current release
+
+  FLASK_ADMIN_SWATCH
```
Comment (hasansezertasan) [inline doc/advanced.rst:555]: Version 2 doesn't have this anymore.

File: doc/advanced.rst [hunk 2]
```diff
@@ -545,3 +545,20 @@ While the wrapped function should accept only one parameter - `ids`::
...
+    Theme for bootswatch in order to customize you UI. See https://bootswatch.com/
+    Example : FLASK_ADMIN_SWATCH = "simplex"
+
+  FLASK_ADMIN_FLUID_LAYOUT
```
Comment (hasansezertasan) [inline doc/advanced.rst:559]: Version 2 doesn't have this anymore.

**Review by hasansezertasan — CHANGES_REQUESTED** [review]

**Comment by hasansezertasan** [comment]
> Thanks for the suggested improvement to document env var configuration on Flask-Admin; I agree this is something we're missing at the moment.
> 
> We've just removed env var configuration for theming (swatch, fluid layout) and move it to a Theme config object passed to the admin instance. It looks like we still have three possible env vars: `ADMIN_RAISE_ON_INTEGRITY_ERROR`, `ADMIN_RAISE_ON_VIEW_EXCEPTION` and `FLASK_ADMIN_SEARCH_SIZE_MAX`. It would be good to add documentation for those. I think it makes sense to flag these at the bottom of 'introduction.rst' rather than hiding them in 'advanced.rst'.
> 
> If you wanted to update the PR I'd be very grateful - if not I'll try to get around to it soon.

Looks like we have a lot more...

https://github.com/pallets-eco/flask-admin/blob/68bc6de4056c493a44b616e9bc4ac7850c0441fe/flask_admin/templates/bootstrap4/admin/lib.html#L259-L285

**Comment by samuelhwilliams** [comment]
🥲  We should have a consistent namespace for all of these, too.

**Comment by TheoLechemia** [comment]
Maybe I would take a look when the 2.0 will be almost stabilised ? 
I see there is still a `MAPBOX_MAP_ID` which is discuss here : https://github.com/pallets-eco/flask-admin/issues/2034

**Comment by samuelhwilliams** [comment]
Thanks for flagging #2034 - agree that we should fix that interface. I've assigned myself to it and will think about what we want to do there.

For now I'm going to close this PR as I'll put one up shortly that rationalises all of the config variables under a FLASK_ADMIN_ namespace and adds some documentation.

**Pull request closed without merging**